app/lib/theme/components/FileComponent.tsx

```diff
@@ -209,23 +238,79 @@ const FileComponent: React.FC<FileComponentProps> = ({
         {error && <ErrorMessage error={error} fileTypes={fileTypes} />}
       </StyledDetails>
       <StyledDetails>{statusLabel}</StyledDetails>
-      <div>
-        <StyledButton
-          addBottomMargin={wrap}
-          id={`${id}-btn`}
-          onClick={(e: React.MouseEvent<HTMLInputElement>) => {
-            e.preventDefault();
-            handleClick();
-          }}
-          variant={buttonVariant}
-          disabled={loading || disabled}
-        >
-          {loading ? (
-            <LoadingSpinner color={isSecondary ? '#000000' : '#fff'} />
-          ) : (
-            buttonLabel()
-          )}
-        </StyledButton>
+      <div
+        style={
+          useFileDate
+            ? { display: 'flex', flexDirection: 'row', alignItems: 'center' }
+            : {}
+        }
+      >
+        {useFileDate && (
+          <div style={{ height: '100%' }}>
+            <h4 style={{ marginBottom: '8px' }}>{`${fileDateTitle}`}</h4>
+            <LocalizationProvider
+              localeText={
+                enUS.components.MuiLocalizationProvider.defaultProps.localeText
+              }
+              dateAdapter={AdapterDayjs}
+            >
+              <StyledDatePicker
+                id={id}
+                sx={getStyles(false)}
+                disabled={false}
+                readOnly={false}
+                onChange={(d: Date | null) => {
+                  if (d) {
+                    const originalDate = new Date(d);
+                    const realDate = new Date(originalDate.toDateString());
```

Comment:
Is this to set it without Hours/Minutes/Seconds? Why don't we just do something like `originalDate.setHours(0,0,0,0)` to zero-out everything. Would read more clear to me

Reply:
I believe this is to handle the time zone stuff and set it in the correct order, it was taken directly from the DatePickerWidget file, only difference is the setter.